src/Routes.js

```diff
@@ -32,6 +33,7 @@ const Routes = (props) => {
                     <Route exact path={paths.sourcesNew} render={() => <SourcesPage />} />
                     <Route path={paths.sourcesEdit} render={() => <SourcesPage />} />
                     <Route path={paths.sourcesRemove} render={() => <SourcesPage />} />
+                    <Route path={paths.sourcesAddApplication} render={() => <SourcesPage />} />
```

Comment:
If you are not adding any additional props to route, just use `component={SourcesPage}` instead of `render`